test/02-functions-test.js

```diff
@@ -67,14 +67,14 @@ describe('internal functions', function () {
     var getModuleDebugId = functions.getModuleDebugId;
     it('names modules based on __filename and package.json', function () {
       expect(getModuleDebugId(__filename))
-        .to.equal('lazy-debug:test:02-functions-test');
+        .to.equal('test:02-functions-test');
       expect(getModuleDebugId(require('./dir1/submodule1')()))
-        .to.equal('lazy-debug:test:dir1:submodule1');
+        .to.equal('test:dir1:submodule1');
       expect(getModuleDebugId(require('./dir1')()))
-        .to.equal('lazy-debug:test:dir1');
+        .to.equal('test:dir1');
       expect(getModuleDebugId(require('./dir1')()))
-        .to.equal('lazy-debug:test:dir1');
```

Comment:
That's what delayed things, finally got time to study what's the issue. It's what I suspected: platform dependent test-case. I'll accept PR as it is and think about the tests after that.